src/test/scala/firrtlTests/AnnotationTests.scala

```diff
@@ -40,20 +41,19 @@ trait AnnotationSpec extends LowTransformSpec {
  * Unstable, Fickle, and Insistent can be tested.
  */
 class AnnotationTests extends AnnotationSpec with Matchers {
-  def getAMap (a: Annotation): AnnotationMap = AnnotationMap(Seq(a))
-  val input: String =
-    """circuit Top :
-       |  module Top :
-       |    input a : UInt<1>[2]
-       |    input b : UInt<1>
-       |    node c = b""".stripMargin
-  val mName = ModuleName("Top", CircuitName("Top"))
-  val aName = ComponentName("a", mName)
-  val bName = ComponentName("b", mName)
-  val cName = ComponentName("c", mName)
+  def getAMap (a: Annotation): Option[AnnotationMap] = Some(AnnotationMap(Seq(a)))
+  def getAMap (as: Seq[Annotation]): Option[AnnotationMap] = Some(AnnotationMap(as))
+  def anno(s: String, value: String ="this is a value"): Annotation =
+    Annotation(ComponentName(s, ModuleName("Top", CircuitName("Top"))), classOf[Transform], value)
```

Comment:
I think I missed that this was in the tests when I originally wrote that comment hahaha